### PR TITLE
Add patch for lbfgsb

### PR DESCRIPTION
**Pull request by Gnimuc**
fix https://github.com/Gnimuc/LBFGSB.jl/issues/9

### CODE REVIEW

**Comment by giordano** [comment]
Sorry, closed by mistake.  You need to rename the directory to match the name of the project as spelled in `build_tarballs.jl`, but we should fix this.

File: L/L_BFGS_B/build_tarballs.jl [hunk 1]
```diff
@@ -3,17 +3,19 @@
 using BinaryBuilder
 
 name = "L_BFGS_B"
-version = v"3.0.0"
+version = v"3.0.1"
```
Comment (giordano) [inline L/L_BFGS_B/build_tarballs.jl]: The upstream source is always the same, why changing this version number?

Comment (Gnimuc) [reply L/L_BFGS_B/build_tarballs.jl]: I think this is just the _jll package version number. Is this rule a convention or a mandatory limitation?

Actually, the patch should be submitted into the upstream, but the upstream seems to be not actively maintained. I bumped the patch number to notify users that this _jll package is a patched version of the upstream.

Comment (giordano) [reply L/L_BFGS_B/build_tarballs.jl]: > I think this is just the _jll package version number. Is this rule a convention or a mandatory limitation?

No, we use the same version number as the upstream package

**Pull request merged**